azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/configuration/LocalizationProvider.kt

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.communication.ui.configuration
+
+internal interface LocalizationProvider {
```

Comment:
Please document interfaces, even internal ones.

Reply:
Even though some existing internal interface for library doesn't include the documentation, I agree its a good practice to include documentation and updated the PR with the requested change.